pkgimage.mk

```diff
@@ -133,3 +133,8 @@ $(eval $(call stdlib_builder,LazyArtifacts,Artifacts Pkg))
 
 $(eval $(call stdlib_builder,SparseArrays,Libdl LinearAlgebra Random Serialization SuiteSparse_jll))
 $(eval $(call stdlib_builder,Statistics,LinearAlgebra SparseArrays))
+
+# precompile Pkg -> REPLExt
+export JULIA_CPU_TARGET=$(JULIA_CPU_TARGET)
+$$(call spawn,$$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg'
+$$(call spawn,$$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg'
```

Comment:
@vchuravy is this enough to get the pkgimages put in the julia shipped depot? I don't see them in there after a build

Reply:
You need to actually define a Makefile rule, something like

```
$(BUILDDIR)/stdlib/REPLExt.release.image: | $(BUILDDIR)/stdlib/Pkg.release.image $(BUILDDIR)/stdlib/REPL.release.image
    $(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg'
```

Reply:
Please feel free to edit this suggestion:
```suggestion
# precompile Pkg -> REPLExt
$$(BUILDDIR)/stdlib/REPLExt.release.image: export JULIA_CPU_TARGET=$(JULIA_CPU_TARGET)
$$(BUILDDIR)/stdlib/REPLExt.debug.image: export JULIA_CPU_TARGET=$(JULIA_CPU_TARGET)

$(BUILDDIR)/stdlib/REPLExt.release.image: | $(BUILDDIR)/stdlib/Pkg.release.image $(BUILDDIR)/stdlib/REPL.release.image
    $(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg'
	$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg'

$(BUILDDIR)/stdlib/REPLExt.debug.image: | $(BUILDDIR)/stdlib/Pkg.debug.image $(BUILDDIR)/stdlib/REPL.debug.image
    $(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no --check-bounds=yes -e 'import REPL, Pkg'
	$(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e 'import REPL, Pkg'
```

Reply:
That looks right, but do note that Makefiles use tabs and not spaces, also I think we need to add `REPLExt` to `independent_stdlibs` in `stdlib/stdlib.mk` so that it get's added to the default set.

Also adding `REPLExt_SRCS :=` and adding a dependency on them

Reply:
I've given it a go but hit this and not sure what's wrong
```
% make -j6
/bin/sh: -c: line 0: unexpected EOF while looking for matching `''
/bin/sh: -c: line 1: syntax error: unexpected end of file
make[1]: *** [stdlib/REPLExt.release.image] Error 2
make: *** [stdlibs-cache-release] Error 2
```